README.md

```diff
@@ -1,35 +1,38 @@
-
-- **Project Management**: Basecamp, Trello, Asana, Flow, Glip, Teamwork
-- **Issue Tracking**: DoneDone
-
-That’s not to say that it doesn’t make sense with other applications, too! These are just the ones who’ve added the Harvest Platform so far.
-
-## How do customers use the time they track via the Harvest Platform?
-It varies by customer. But the two main Harvest features that customers will use with their time tracked via the Harvest Platform are invoicing and reporting. Some customers need to bill their clients for the time they track via your application, whether they’re managing projects or providing customer support. Other customers just use this time data internally to see where their time is being spent, gauge profitability, or monitor employee performance.
-
-## What should you know before working on implementing the Harvest Platform with your app?
-- In our technical documentation, we say that the [permalink](https://github.com/harvesthq/platform/blob/master/Platform_Technical_FAQ.md#what-configuration-options-do-i-need-to-specify) is optional. Which is true, it is. But, linking Harvest time entries back to your app’s attributes is one of the top reasons why customers use the Harvest Platform integrations. Having this linkage makes the success of your integration all the more likely.
-- We say that you can apply custom styles to the Harvest timer within your app, but we recommend that you use blue as the color to indicate a running timer. Our customers are used to this in the Harvest interface, so it won’t require any additional thought to understand when they have a timer running from your app.
-
-**Ready to dive into the few technical details?** Click [here](https://github.com/harvesthq/platform/blob/master/Platform_Technical_FAQ.md) for a technical FAQ or
-[watch the video introduction](http://www.youtube.com/watch?v=5_fVPnH7VBU).
+# Integrating with Harvest
+
+<table>
+  <thead>
+    <tr>
+      <th>Harvest Button</th>
+      <th>Harvest Widget</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td width="50%">
+        <a href="button.md"><img src="http://cl.ly/image/0q0W1H003p1n/Image%202015-06-15%20at%204.00.36%20PM.png" alt="Screenshot of the Harvest Button in action"></a>
+      </td>
+      <td width="50%">
+        <a href="widget.md"><img src="http://cl.ly/image/0a1q151i2s0b/Image%202015-06-15%20at%204.14.54%20PM.png" alt="Screenshot of the Harvest Widget in action"></a>
+      </td>
+    </tr>
+    <tr>
+      <td>The Harvest Button is a script that runs within your application to provide a clock icon that, when clicked, shows a modal dialog with a form to track time in Harvest. <a href="button.md">Try it now…</a></td>
+      <td>The Harvest Widget is a frame that is loaded by your application that contains a form to track time in Harvest. <a href="widget.md">Try it now…</a></td>
```

Comment:
I'd change the `…` to an exclamation mark! Feels better to me as a call to action.